django_fsm_log/apps.py

```diff
@@ -9,6 +9,7 @@
 class DjangoFSMLogAppConfig(AppConfig):
     name = 'django_fsm_log'
     verbose_name = "Django FSM Log"
+    default_auto_field = 'django.db.models.BigAutoField'
```

Comment:
This overrides user/default settings then, is this intended?

Reply:
I was under the impression that's this is the only backward compatible choice.
What do you suggest we do otherwise ?

Reply:
Projects should configure it.